### ISSUE TITLE
Issue with display of Pareto optimal solution 

### ISSUE THREAD

**Issue by yinzihao95** [Bardbo/geatpy-Multi-objective-Optimization-for-Shortest-Path#1]
The display for the Pareto optimal solution provide in ur code only display five point which is optimal solution. I think there is something wrong, can you please tell how we can have the code display of the solution point in the 3D visualization.

**Comment by Bardbo** [comment]
Yes, there are only five Pareto optimal solutions in this problem. May I ask where you think there is a problem? Can you expand on it?

Regarding the issue of 3D visualization, since the objective function in the example is 3, setting the drawing property to 1 will result in a 3D result graph

In addition, the geatpy version used here is 2.6, and the current latest version is 2.7. It is recommended to use 2.7

(Please forgive my poor English, it was translated through translation software)

**Comment by Bardbo** [comment]
> The display for the Pareto optimal solution provide in ur code only display five point which is optimal solution. I think there is something wrong, can you please tell how we can have the code display of the solution point in the 3D visualization.

Yes, there are only five Pareto optimal solutions in this problem. May I ask where you think there is a problem? Can you expand on it?

Regarding the issue of 3D visualization, since the objective function in the example is 3, setting the drawing property to 1 will result in a 3D result graph

In addition, the geatpy version used here is 2.6, and the current latest version is 2.7. It is recommended to use 2.7

(Please forgive my poor English, it was translated through translation software)
